postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -486,6 +575,52 @@ def _explain_pg_stat_activity(self, rows):
                     tags=self._tags + ["error:collect-plan-for-statement-crash"] + self._check._get_debug_tags(),
                     hostname=self._check.resolved_hostname,
                 )
+        return events
+
+    def _create_activity_event(self, rows, active_connections):
+        self._time_since_last_activity_event = time.time()
+        active_sessions = self._to_active_sessions(rows)
+        if len(active_sessions) > self._activity_max_rows:
+            self._check.gauge(
+                "dd.postgres.create_activity_event.active_sessions_pre_truncate.size",
```

Comment:
Do we need this? I think we should have the total number of connections from `PG_ACTIVE_CONNECTIONS_QUERY` and comparing that to how many rows come in we should be able to see how much was truncated?

Reply:
sure we can emit a monitoring metric in the backend instead